region/brinstar/green/Green Brinstar Main Shaft.json

```diff
@@ -2645,6 +3057,62 @@
       ],
       "flashSuitChecked": true
     },
+    {
+      "link": [5, 7],
+      "name": "Come in Shinecharging, Leave With Temporary Blue (Full Runway)",
+      "entranceCondition": {
+        "comeInShinecharging": {
+          "length": 12,
+          "openEnd": 0
+        }
+      },
+      "requires": [
+        "canLongChainTemporaryBlue",
+        "canXRayTurnaround",
+        {"ammo": {"type": "PowerBomb", "count": 1}}
+      ],
+      "exitCondition": {
+        "leaveWithTemporaryBlue": {}
+      },
+      "unlocksDoors": [{"types": ["ammo"], "requires": []}]
+    },
+    {
+      "link": [5, 7],
+      "name": "Come in Shinecharging, Leave With Temporary Blue (Partial Runway)",
+      "entranceCondition": {
+        "comeInShinecharging": {
+          "length": 5,
+          "openEnd": 0
+        }
+      },
+      "requires": [
+        "canLongChainTemporaryBlue",
```

Comment:
For placement sure, but this is 2 temp blues.
Note about falling through the shot block - wait to morph until you are through and then you can bounce on the single tile platform to reach the door.

Reply:
It was for placement, yeah, because I had a hard time with blindly unmorphing onto that single tile platform. But with your note, it becomes a lot easier. Added that and changed the requirement to just "canChainTemporaryBlue".